quiz-application/src/main/java/com/example/quiz/Game.java

```diff
@@ -1,36 +1,78 @@
 package com.example.quiz;
 
-import com.google.gson.Gson;
-import com.google.gson.reflect.TypeToken;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import kong.unirest.Unirest;
 import org.springframework.shell.standard.ShellComponent;
 import org.springframework.shell.standard.ShellMethod;
+import org.springframework.shell.standard.ShellOption;
 
+import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
 import java.util.stream.Collectors;
 
 import static java.lang.System.lineSeparator;
+import static java.util.Objects.isNull;
 
 @ShellComponent
 public class Game {
 
+    private static final List<String> OPTIONS = List.of("trivia", "countries");
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
     List<QuestionDTO> questions;
     QuestionDTO currentQuestion;
-    int count = 5;
+    int questionsNumber = 5;
+    int count = questionsNumber;
+    Integer options;
+    char quizType;
 
     @ShellMethod("Starts game")
-    public String start() {
-        questions = new Gson().fromJson(
-                Unirest.get("https://the-trivia-api.com/api/questions")
-                        .queryString("limit", 5)
-                        .asJson()
-                        .getBody()
-                        .getArray()
-                        .toString(), new TypeToken<List<QuestionDTO>>() {
-                }.getType());
+    public String start(@ShellOption(defaultValue = "") String option, @ShellOption(defaultValue = "") String answerOptions) throws JsonProcessingException {
+        if(count != questionsNumber) {
+            count = questionsNumber;
+        }
+        //add support for single letter menu option #jira QUIZ-334
+        if (option.isBlank() ||
+                (isNull(OPTIONS.stream().filter(o -> o.equalsIgnoreCase(option)).findAny().orElse(null)) &&
+                isNull(OPTIONS.stream().map(o -> String.valueOf(o.charAt(0))).filter(o -> o.equalsIgnoreCase(option)).findAny().orElse(null)))) {
+            return "PLEASE PROVIDE QUIZ TYPE\nAVAILABLE OPTIONS ARE\ntrivia (or for shortcut t)\ncountries (or for shortcut c) + optional num of answers from 2-5";
```

Comment:
message - wynosimy do stałych w osobnym pliku 
dobrze byłoby usunąć zahardkodowane opcje a przekazac  opcje gry  wyniesione wczesniej np do enuma 

tak naprawdę to nie jest prawdą że `optional num of answers from 2-5` - zahardkodowaliśmy `int questionsNumber = 5;` więc albo zmieniamy msg - albododajemy parametr numberOfAnswers